source/_docs/z-wave/controllers.markdown

```diff
@@ -13,7 +13,9 @@ This Z-Wave integration is deprecated and replaced with a [new implementation ba
 
 You need to have a compatible Z-Wave stick or module installed. This needs to be a *static controller*, which most Z-Wave sticks and modules will be. If yours is a *bridge* device then it won't work with [OpenZWave](http://openzwave.com/), which is what provides Home Assistant's Z-Wave capabilities. The following devices have been confirmed to work:
 
-- Aeotec Z-Stick Series 5 (see note below)
+- Aeotec Z-Stick Gen5 (see note below)
+- Aeotec Z-Stick 7
+- Aeotec Z-Pi 7
```

Comment:
Are these compatible with OpenZWave as well?

Reply:
Z-Stick 7 and Z-Pi7  are both bridge controllers, they won't work with OpenZWave.

Reply:
Well this package is for OpenZWave as well, so that makes this change invalid.

Reply:
Got it, thank you. Didn't realize that since it mentions info about Bridge controllers not being for OpenZWave. I'll close this one up.

Reply:
@frenck - The ZST10 is a Bridge controller as well which is why we added the Z-Stick 7 and Z-Pi 7 to this list: https://products.z-wavealliance.org/products/4260?selectedFrequencyId=-1